pipeline/frontend/yaml/compiler/convert.go

```diff
@@ -16,7 +16,7 @@ import (
 	yaml_types "github.com/woodpecker-ci/woodpecker/pipeline/frontend/yaml/types"
 )
 
-func (c *Compiler) createProcess(name string, container *yaml_types.Container, section string) *backend_types.Step {
+func (c *Compiler) createProcess(name string, container *yaml_types.Container, typ backend_types.StepType) *backend_types.Step {
```

Comment:
```suggestion
func (c *Compiler) createProcess(name string, container *yaml_types.Container, type backend_types.StepType) *backend_types.Step {
```

Reply:
this was intentional to not overload the `type` keyword ... but I can rename it to stepType I guess